### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by alexeagle**
document our versioning scheme

came up in #427

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -37,6 +37,14 @@ to try it you should expect to spend some time debugging and reporting bugs.
 
 ## Usage
 
+### Versioning
+
+Tsickle uses lots of TypeScript compiler APIs, so you must use a tsickle version that matches your TypeScript version.
+We publish our releases with the minor version corresponding to the TypeScript version it works with, so eg.
+`0.2.6` supports TypeScript 2.0, `0.21.6` supports TypeScript 2.1, etc.
```
Comment (evmar) [inline README.md]: Is this a typo?  What version will we use for TypeScript 21.6?

Comment (kylecordes) [inline README.md]: Perhaps the ability to encode times as JSDoc+Closure will end up "in the box" in TypeScript, somewhere between now and 21.6...

**Comment by mprobst** [comment]
@alexeagle ping.

File: README.md (hunk 1 above)
Comment (alexeagle) [inline README.md]: Not a typo, that was the last 0.21.* release. LG?

Comment (evmar) [inline README.md]: What version will we use for TypeScript 21.6?

Comment (alexeagle) [inline README.md]: I dunno, I'm just trying to document what we have now. We'll be in trouble and will have to change our numbering scheme?

**Comment by evmar** [comment]
ping, is this obsolete?

**Pull request closed without merging**